flow/model/qvalue/etime.go

```diff
@@ -15,26 +15,26 @@ const (
 
 type ExtendedTime struct {
 	time.Time
-	NestedKind NestedKind
+	NestedKind ExtendedTimeKind
```

Comment:
do we even need this ExtendedTimeKind, maybe this whole struct should be removed.

Reply:
removed